src/main/java/org/osiam/resources/controller/ResourceController.java

```diff
@@ -64,4 +74,29 @@
         result.add("id");
         return result;
     }
+
+    protected MappingJacksonValue buildWrapper(T resource, String attributes) {
```

Comment:
Well, we are giving up type safety, I'm not very fond of that. However it's probably ok in this case and I will follow your advice here.